docs/Parameters.rst

```diff
@@ -1275,7 +1275,9 @@ LightGBM supports weighted training. It uses an additional file to store weight
     0.8
     ...
 
-It means the weight of the first data row is ``1.0``, second is ``0.5``, and so on.
+It means the weight of the first data row is ``1.0``, second is ``0.5``, and so on. Weight should be non-negative
+values because the Hessian value multiplied by weight is supposed to be non-negative.
```

Comment:
Thank you for dong this. I think it would be enough to say `weighs should be non-negative`. Exposing details about Hessian values may make the users confusing if they don't understand how Hessian values are used in GBDT algorithms.

Reply:
Ok, here the simple explanation works because you can already infer it from the example.